docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -1,25 +1,23 @@
 [[set-up-driver-plugin]]
-= Setting up the driver plugin
+= Setting Up the Driver Plug-in
 :icons:
 :toc: macro
 :toc-title:
 :toclevels: 1
 
 toc::[]
 
-[[hypervisor-drivers-overview]]
+[[setup-driver-plugin-overview]]
```

Comment:
Thanks, and actually even overview anchor IDs need to be unique across the entire repository (as in no matter the hierarchy). This is according to the openshift guidelines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#unique-ids